tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -89,21 +96,22 @@ def copy_granule_between_buckets(
     file_versions = s3.list_object_versions(
         Bucket=destination_bucket, Prefix=destination_key
     )
-    for ver in file_versions["Versions"]:
-        if ver["IsLatest"]:
-            LOGGER.info("collecting metadata from file version")
-            etag = ver["ETag"]
-            sizeInBytes = ver["Size"]
-            version = ver["VersionId"]
-            break
+    latest_version = next(
```

Comment:
Added a comment. This is functionally identical to the loop that was there before, but I feel that using a loop to look for one item is a bit weird. Especially since it makes the compiler think that the value might not be found.